spec/features/post_peep_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'posting peeps' do
+  scenario 'registered users can post peeps' do
+    visit '/peeps/new'
+
+    fill_in('new_peep', with: 'Betis - Real Madrid today!')
+    click_button 'Publish peep'
+
+    expect(page).to have_content 'Betis - Real Madrid today!'
+  end
+end
```

Comment:
The counterpoint to this is that "an unregistered / signed out user cannot post peeps".  This is also worth testing.